src/db_client/builder.rs

```diff
@@ -1,5 +1,7 @@
 // Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
 
+//! Client builder
+
```

Comment:
remove mode `Cluster ` and `Standalone` for builder client

Reply:
Maybe we should rename the two mode to `Direct` and `Proxy`.